### PR TITLE
plume: handle non-existent fcos releases.json

### PR DESCRIPTION
**Pull request by mike-nguyen**
Use an empty json instead of failing when there is no releases.json

### CODE REVIEW

**Comment by arithx** [comment]
Closes #1022

File: cmd/plume/release.go [hunk 1]
```diff
@@ -492,16 +493,30 @@ func modifyReleaseMetadataIndex(spec *fcosChannelSpec, commitId string) {
...
 
 	path := filepath.Join("prod", "streams", specChannel, "releases.json")
-
+	var data []byte
 	f, err := api.DownloadFile(spec.Bucket, path)
 	if err != nil {
-		plog.Fatalf("downloading release metadata index: %v", err)
-	}
-	defer f.Close()
+		printErr := false
```
[the remark this answers is not in the document]
Comment (mike-nguyen) [reply cmd/plume/release.go]: Thanks @arithx. Option 2 is pretty clever and golang-eque!

**Comment by mike-nguyen** [comment]
Pushed requested changes :arrow_up:

**Comment by mike-nguyen** [comment]
ok to merge

**Comment by mike-nguyen** [comment]
thanks for the feedback @arithx!

**Pull request merged**